### PR TITLE
Impl: Validate that eth_gasEstimate works for writing to contract storage slot with delegate call 

### PR DESCRIPTION
**Pull request by ivanparnarev**
**Description**:
Create a test with eth_gasEstimate invoking a solidity function, which modifies the storage of a test smart contract with delegate call. Validate that the slot modification passes successfully and the new value is written and an estimated gas is returned.

**Checklist**

- [x] HederaDelegateCallOperation implemented
- [x] Tested ('ethGasEstimateForWritingToContractStorageWithDelegateCallWorks')
- [x] EthCall.sol updated with the new functions (as well as bytecodes)
- [x] DelegateCallTest.sol added for test purposes (all bytecodes are updated)

### CODE REVIEW

File: hedera-mirror-web3/src/test/resources/insert_mock_data.sql [hunk 1]
```diff
@@ -43,9 +43,9 @@ VALUES (4, 1253, 0, 0, int8range(12235235, 32323436), 'TOKEN',
...
...102f8576102f76101f8565b5b6103038482856102ba565b509392505050565b600082601f8301126103205761031f6101f3565b5b81356103308482602086016102c9565b91505092915050565b600080604083850312156103505761034f61018b565b5b600061035e858286016101de565b925050602083013567ffffffffffffffff81111561037f5761037e610190565b5b61038b8582860161030b565b9150509250929050565b600081519050919050565b600082825260208201905092915050565b60005b838110156103cf5780820151818401526020810190506103b4565b838111156103de576000848401525b50505050565b60006103ef82610395565b6103f981856103a0565b93506104098185602086016103b1565b610412816101fd565b840191505092915050565b6000602082019050818103600083015261043781846103e4565b905092915050565b600081519050919050565b600081905092915050565b60006104608261043f565b61046a818561044a565b935061047a8185602086016103b1565b80840191505092915050565b60006104928284610455565b915081905092915050565b7f46756e6374696f6e2063616c6c20726576657274656400000000000000000000600082015250565b60006104d36016836103a0565b91506104de8261049d565b602082019050919050565b60006020820190508181036000830152610502816104c6565b905091905056fea26469706673582212201409b71eee75fc8d754d7cdacbe84dec1804bf43cb529a81400834105757734764736f6c634300080f0033', 'hex'));
```
Comment (IvanKavaldzhiev) [inline hedera-mirror-web3/src/test/resources/insert_mock_data.sql:46]: Do we need file_id?

File: hedera-mirror-web3/src/test/resources/insert_contract.sql [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
...102f8576102f76101f8565b5b6103038482856102ba565b509392505050565b600082601f8301126103205761031f6101f3565b5b81356103308482602086016102c9565b91505092915050565b600080604083850312156103505761034f61018b565b5b600061035e858286016101de565b925050602083013567ffffffffffffffff81111561037f5761037e610190565b5b61038b8582860161030b565b9150509250929050565b600081519050919050565b600082825260208201905092915050565b60005b838110156103cf5780820151818401526020810190506103b4565b838111156103de576000848401525b50505050565b60006103ef82610395565b6103f981856103a0565b93506104098185602086016103b1565b610412816101fd565b840191505092915050565b6000602082019050818103600083015261043781846103e4565b905092915050565b600081519050919050565b600081905092915050565b60006104608261043f565b61046a818561044a565b935061047a8185602086016103b1565b80840191505092915050565b60006104928284610455565b915081905092915050565b7f46756e6374696f6e2063616c6c20726576657274656400000000000000000000600082015250565b60006104d36016836103a0565b91506104de8261049d565b602082019050919050565b60006020820190508181036000830152610502816104c6565b905091905056fea26469706673582212201409b71eee75fc8d754d7cdacbe84dec1804bf43cb529a81400834105757734764736f6c634300080f0033', 'hex'));
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply hedera-mirror-web3/src/test/resources/insert_contract.sql:22]: at least 1 with file_id for coverage is probably good.

**Review by Nana-EC — APPROVED** [review]
LG

File: hedera-mirror-web3/src/test/resources/insert_mock_data.sql [hunk 3]
```diff
@@ -43,13 +43,19 @@ VALUES (4, 1253, 0, 0, int8range(12235235, 32323436), 'TOKEN',
...
```
Comment (IvanKavaldzhiev) [inline hedera-mirror-web3/src/test/resources/insert_mock_data.sql:46]: file_id probably should be 1257?

File: hedera-mirror-web3/src/test/resources/insert_contract.sql [hunk 4]
```diff
@@ -1,30 +1,35 @@
...
```
Comment (IvanKavaldzhiev) [inline hedera-mirror-web3/src/test/resources/insert_contract.sql:22]: file_id probably should be 1257?

**Pull request merged**